### ISSUE TITLE
Issue from ÉTS Mobile 

### ISSUE THREAD

**Issue by clubapplets-server** [ApplETS/Notre-Dame#294]
**Describe the issue** 
``````

**Screenshot** 
![screenshot](https://github.com/ApplETS/Notre-Dame-Bug-report/blob/main/bugPicture-774439157.png?raw=true)

**Device Infos** 
- **Device:** iPhone8,1 
- **Version:** 4.4.8 
- **Connectivity:** ConnectivityResult.wifi 
- **Build number:** 1634853831 
- **Platform operating system:** ios 
- **Platform operating system version:** 14.6

**Comment by MysticFragilist** [comment]
This has nothing in it, closing

**Comment by apomalyn** [comment]
Linked to #88 will be solved #293.
